### PR TITLE
{phys}[foss/2020b,intel/2020b] PSolver v1.8.3

### PR DESCRIPTION
**Pull request by micaeljtoliveira**
This version has a new dependency on the futile library, so we also include futile-1.8.3-foss-2020b and futile-1.8.3-foss-2020b.

### CODE REVIEW

File: easybuild/easyconfigs/f/futile/futile_intel_bug.patch [hunk 1]
```diff
@@ -0,0 +1,47 @@
+diff -rup futile-1.8.3.orig/flib/fpython.f90 futile-1.8.3/flib/fpython.f90
```
Comment (Micket) [inline easybuild/easyconfigs/f/futile/futile_intel_bug.patch:1]: Could you please add a short description what the patch does, and note who the author is?

**Review by Micket — DISMISSED** [review]

File: easybuild/easyconfigs/f/futile/futile_intel_bug.patch (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/f/futile/futile_intel_bug.patch:1]: @micaeljtoliveira Ping on this?

Comment (micaeljtoliveira) [reply easybuild/easyconfigs/f/futile/futile_intel_bug.patch:4]: @boegel Thanks for the ping. For some reason I missed this comment. I've now added a short description and the author to the patch.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
node3100.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/1828d3932a4b751a131fb09d6a6c7501 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
node3537.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/fd41eb49b5583eba99a7ea7ae846210b for a full test report.

**Comment by boegel** [comment]
Going in, thanks @micaeljtoliveira!

**Pull request merged**